client/my-sites/domains/domain-management/edit/domain-types/renew-options-row.jsx

```diff
@@ -0,0 +1,160 @@
+	justify-content: space-between;
+	width: 100%;
+
+	@media ( min-width: 550px ) {
+		flex-direction: row;
+	}
+`;
+
+const Section = styled.div`
+	flex: 1;
+	margin: 10px;
+`;
+
+const Header = styled.div`
+	text-transform: uppercase;
+	font-size: 12px;
+	color: var( --color-neutral-50 );
+`;
+
+const BodyText = styled.div`
+	display: flex;
+	align-items: center;
+	margin: 10px 0;
+
+	& > svg {
+		padding-right: 5px;
+	}
+
+	& > span {
+		font-size: 12px;
+	}
+`;
+
+const AutoRenew = styled( BodyText )`
+	color: var( --color-success );
+
+	& > svg {
+		fill: var( --color-success );
+	}
+`;
+
+const ManualRenew = styled( BodyText )`
+	font-weight: 600;
+
+	& > svg {
+		fill: var( --color-warning-30 );
+	}
+`;
+
+function RenewOptionsRow( { domain, isLoadingPurchase, moment, purchase, selectedSite } ) {
+	const [ renewalText, setRenewalText ] = useState( null );
+	const [ autoRenewStatus, setAutoRenewStatus ] = useState( null );
+	const translate = useTranslate();
+
+	useEffect( () => {
+		if ( ! purchase ) return;
+
+		if ( 'manualRenew' !== purchase.expiryStatus ) {
+			setAutoRenewStatus( 'Enabled' );
+			setRenewalText(
+				translate( 'This domain will automatically renew on %(renewal_date)s.', {
+					args: {
+						renewal_date: moment.utc( purchase.renewDate ).format( 'LL' ),
+					},
+				} )
+			);
+		} else {
+			setAutoRenewStatus( 'Disabled' );
+			setRenewalText( null );
+		}
+	}, [ domain, purchase ] );
+
+	if ( isLoadingPurchase || ! selectedSite.ID ) {
+		return (
+			<div className="domain-types__expires-placeholder">
+				<p />
+			</div>
+		);
+	}
+
+	let expirationText;
+	const renewalPrice = getRenewalPrice( purchase );
+	const currencyCode = purchase.currencyCode;
+	const formattedPrice = formatCurrency( renewalPrice, currencyCode, { stripZeros: true } );
+
+	if ( domain.type === domainTypes.MAPPED && ! domain.expiry ) {
+		expirationText = translate( 'Never' );
+	} else {
+		expirationText = moment.utc( domain.expiry ).format( 'LL' );
+	}
+
+	return (
+		<ParentDiv>
+			<Section>
+				<Header>{ translate( 'Renewal price' ) }</Header>
+				<BodyText>{ formattedPrice }/year</BodyText>
+			</Section>
+			<Section>
+				<Header>{ translate( 'Auto renewal' ) }</Header>
+				{ 'Enabled' === autoRenewStatus ? (
+					<AutoRenew>
+						<MaterialIcon icon="check_circle" />
+						{ autoRenewStatus }
+					</AutoRenew>
+				) : (
+					<ManualRenew>
+						<Gridicon icon="notice" />
+						{ autoRenewStatus }
+					</ManualRenew>
+				) }
+				<BodyText>
+					<span>{ renewalText }</span>
+				</BodyText>
+				{ domain.currentUserCanManage && (
+					<AutoRenewToggle
+						planName={ selectedSite.plan.product_name_short }
+						purchase={ purchase }
+						siteDomain={ selectedSite.domain }
+						toggleSource="registered-domain-status"
+						withTextStatus={ true }
+					/>
+				) }
+			</Section>
+			<Section>
+				<Header>{ translate( 'Expiration date' ) }</Header>
+				<BodyText>{ expirationText }</BodyText>
+				<RenewButton
```

Comment:
The button text is different from the mockup. But I'm okay if you think `Renew now for $xx` is better than `Renew manually for a year` as you're the marketing expert in the team. 😄